src/docs/get-started/install/_get-sdk-linux.md

```diff
@@ -134,9 +135,8 @@ command again to verify that you’ve set everything up correctly.
 {% include_relative _analytics.md %}
 
 [Flutter repo]: {{site.github}}/flutter/flutter
+[install Flutter using the Snap Store]: https://snapcraft.io/store
```

Comment:
It would be better to link directly to the store page for the flutter snap, which would provide an install button.

https://snapcraft.io/flutter

Reply:
Done